.mage/magefile.go

```diff
@@ -31,8 +31,8 @@ import (
 )
 
 func init() {
-	mgmake.GenerateMakefiles(
-		mgmake.Makefile{
+	gen.GenerateMakefiles(
```

Comment:
IMO we should find a better package name for this public API

Reply:
I reverted to the old package name